tests/dry_run.yaml

```
@@ -26,7 +26,7 @@ create: # Configuration for the create subcommand
   data:
     db: 'test'
     run_in_start_at: 7010
-    run_out_start_at: 8010
+    run_out_start_at: 7010
```

Did you mean to change this?

```suggestion
    run_out_start_at: 8010
```

---

Yes that's intended, to test the plotting library (which will otherwise complain)